packages/editor-ui/src/components/RunDataTable.vue

```diff
@@ -199,9 +200,18 @@ export default mixins(externalHooks).extend({
 			draggedColumn: false,
 			draggingPath: null as null | string,
 			hoveringPath: null as null | string,
+			showMappingIntroHint: false,
```

Comment:
nitpick: Naming is a bit confusing here. I think `mappingHintVisible` would make more sense.

Reply:
makes sense.. ya I struggled a bit here with that. will update